Makefile

```diff
@@ -4,9 +4,6 @@ all:
 flash:
 	west flash
 
-nucleo:
-	west build -b nucleo_f446re app -DSHIELD=launch_mikroe -p
```

Comment:
Why was this removed?

Reply:
> The use case for app_nucleo is typically an ad-hoc app to test some specific functionality, so a directory full of such samples is better than a single app that keeps changing version-to-version or branch-to-branch.

If this line in my PR is wrong then I'll unstage, but that's my thinking

Reply:
Would like to keep for the time being, but I do agree

Reply:
No HEAD has an `app_nucleo` right now though